zebra/zebra_vxlan.c

```diff
@@ -826,7 +826,7 @@ static int zvni_map_to_svi_ns(struct ns *ns,
 	struct interface **p_ifp = (struct interface **)_p_ifp;
 	struct zebra_if *zif;
 
-	if (!in_param)
+	if (!in_param || !p_ifp)
```

Comment:
Let's actually optimize the code in a spot that matters, imo zvni_map_to_svi is the only place that this is ever called and we never pass in a non double pointer( nor would it ever make sense to not do so )  What are we actually trying to do here?

Reply:
Fixed.